lib/chat_message.js

```diff
@@ -271,9 +272,9 @@ class ChatMessage {
       const format = lang[this.translate]
       if (!format) message += args.join('')
       else message += vsprintf(format, args)
-    }
+    } else if (this.translate) message += lang[this.translate]
```

Comment:
Translate in advancement dont get translated so, i fix it
![Result](https://user-images.githubusercontent.com/18616827/86471523-bbe74b00-bd67-11ea-8840-94ba1bfa981e.png)